### ISSUE TITLE
update kubeadm cgroup driver documentation once KubeletCgroupDriverFromCRI goes GA

### ISSUE THREAD

**Issue by neolit123** [kubernetes/kubeadm#3004]
original k/k ticket:
- https://github.com/kubernetes/kubernetes/issues/99808

the new feature gate was added in a previous release and is planned for beta in 1.31:
- https://github.com/kubernetes/enhancements/issues/4033

once GA we should remove this page:
- https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/

perhaps SIG node and feature owners will update:
- https://kubernetes.io/docs/setup/production-environment/container-runtimes/#cgroup-drivers

users would still be required to pick a driver and configure it on the side of container runtimes (e.g. containerd).

**Comment by pacoxu** [comment]
> the new feature gate was added in a previous release and is planned for beta in 1.31:
> [kubernetes/enhancements#4033](https://github.com/kubernetes/enhancements/issues/4033)

This is planned to be beta in this release cycle. 
- We may use it if the FG is enabled and the cgroup driver can be detected from the CRI. And we can fall back to our previous logic if not.

**Comment by neolit123** [comment]
> > the new feature gate was added in a previous release and is planned for beta in 1.31:
> > [kubernetes/enhancements#4033](https://github.com/kubernetes/enhancements/issues/4033)
> 
> This is planned to be beta in this release cycle.
> 
> * We may use it if the FG is enabled and the cgroup driver can be detected from the CRI. And we can fall back to our previous logic if not.

i don't think we have any remaining logic for this, do we? we just set the cgroup driver to systemd.

**Comment by neolit123** [comment]
KubeletCgroupDriverFromCRI went BETA in 1.31
https://github.com/kubernetes/kubernetes/pull/125828/

**Comment by pacoxu** [comment]
https://github.com/kubernetes/enhancements/issues/4033#issuecomment-2318800589
- this went to beta in v1.31 and has no plan in v1.32.

**Comment by afbjorklund** [comment]
* docker https://github.com/Mirantis/cri-dockerd/pull/425 not merged yet (in 0.3.16)

**Comment by pacoxu** [comment]
https://github.com/kubernetes/kubeadm/issues/3229 is related to this one.

**Comment by neolit123** [comment]
will update this as part of 
- https://github.com/kubernetes/website/pull/55979

**Comment by carlory** [comment]
@neolit123  The fallback is still supported in 1.37. Should we update the doc after https://github.com/kubernetes/kubernetes/pull/135923 is merged? That PR should be merged in 1.38.
